### ISSUE TITLE
Change the flayout icon color

### ISSUE THREAD

**Issue by AlexeyZhukoff** [dotnet/maui#6738]
### Description

How can i change flyout icon color? It's always blue, but in orinal file i have black color

### Steps to Reproduce

1. Download example app
2. Run it
3. You see blue flyout icon
[Archive.zip](https://github.com/dotnet/maui/files/8602916/Archive.zip)
 

### Version with bug

Release Candidate 2 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iOS 15.4

### Did you find any workaround?

no

**Comment by Redth** [comment]
Try setting your own custom image source for the flyout icon?

**Comment by AlexeyZhukoff** [comment]
Hello, in example project i set my custom image for flyout icon, but icon always tint to blue color.

**Comment by AlexeyZhukoff** [comment]
> Try setting your own custom image source for the flyout icon?
Hello, in example project i set my custom image for flyout icon, but icon always tint to blue color.

**Comment by XamlTest** [comment]
Verified this issue with Visual Studio Enterprise 17.3.0 Preview 1.0 [32427.455.main]. Repro on iOS 15.4. Sample Project: [MauiApp2.zip](https://github.com/dotnet/maui/files/8629421/MauiApp2.zip)

**Comment by lnaie** [comment]
I see this happening as well in the latest maui with the default vs2022 preview template. If I keep it pressed eventually there is an indication that the hamburger icon is there. How do I fix its color in the default state?

**Comment by lnaie** [comment]
> I see this happening as well in the latest maui with the default vs2022 preview template. If I keep it pressed eventually there is an indication that the hamburger icon is there. How do I fix its color/visibility in the default state?

Ok, It got fixed in the latest, I've just installed. Thanks!

**Comment by AntonKosenkoDX** [comment]
@Redth Hello! Any update on this issue?

**Comment by Zhanglirong-Winnie** [comment]
Verified this issue with Visual Studio Enterprise 17.6.0 Preview 2.0. Repro on iOS simulator with sample project.
[Archive.zip](https://github.com/dotnet/maui/files/11330338/Archive.zip)

**Comment by Qythyx** [comment]
@Redth , it's approaching 1 year since this was moved to the backlog. Any chance this will be picked up soon?

**Comment by markkellaway** [comment]
> @Redth , it's approaching 1 year since this was moved to the backlog. Any chance this will be picked up soon?

+1
This issue is causing us headaches, having to use the default hamburger menu is not a good workaround for us.

**Comment by HobDev** [comment]
Facing this issue on Windows. I am developing with .NET 8 RC2

**Comment by Thaldoras** [comment]
We use a FlyoutPage that has a TabbedPage as Detail.
Inside each Tab is a NavigationPage.

To change the color of the back button in a NavigationPage you can use
`NavigationPage.SetIconColor(this, AppColours.PrimaryDark);`

We make that call in the base class for all out pages. By some kind of black magic this will update the color of the burger menu icon (flyout menu icon) on Android. 

For iOS it will not update the color of the flyout icon until you have first navigated to a page inside on the tabs (a page that displays the back button) and then navigated back. 

TLDR: if you have a NavigationPage inside your FlyoutPage detail then you can use the above method to change the icon color.

**Comment by Thaldoras** [comment]
Following up. You can use the NavagationPage BarTextColor property to set the color of the icon.

**Comment by MauiUIui** [comment]
This seems like a simple issue to solve, yet nobody is doing anything about it for two years? Really? This effects the design greatly.

Well, in case it is not clear here is a summery that might help fixing:
When no custom FlyoutIcon is selected, the color can be changed by changing the `Shell.ForegroundColor`, which by the way also effects the color of the `BackButton`.
However as soon as you want to have your own HamburgerMenu look, which I assume almost everybody would want, you have no control over the color, as it apparently becomes white (on my android at least, where the android specific **colorPrimary**, **colorPrimaryDark** and
**colorAccent** colors aren't set to white.
My proposals for a solution:
1- Do you thing by correcting this, probably by making a property (preferably of type Brush) to control this color.
2- Change the type of the `Shell.FlyoutIcon` property from `ImageSource` to `Image` so that we could use the community toolkit's `IconTintColorBehavior` to change the `TintColor` (not perfect, but would work for most people).
3- Keep ignoring us and let's die using the dull default hamburger menu image, just so we could control the color and not break our UI theme.

**Comment by LeoJHarris** [comment]
👀

**Comment by MauiUIui** [comment]
> **Tested the behaviour in the following platforms**
[ ] Android
[x]  Windows
[x]  iOS
[x] Mac

Does this mean it still is not working on Android? I mean testing on Android should be accomplishable for anyone I would imagine, so I can't understand why one otherwise would test on all platforms and leave Android out, which is currently the platform where MAUI is mostly widely used to target if I'm not mistaking.
